### PR TITLE
Create a building guide + update README.md

### PR DESCRIPTION
**Pull request by lynxnb**

### CODE REVIEW

**Review by PixelyIon — CHANGES_REQUESTED** [review]
Great job, I just have a couple of nits

File: BUILDING.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+> If you opened Android Studio for the first time, choose the `Standard` install type and complete the setup wizard leaving all settings to their default value.
+> <p><img height="400" src="https://user-images.githubusercontent.com/37104290/162196602-4c142ed0-0c26-4628-8062-7ac9785201cc.png" /></p>
+> If you get any errors on "Intel® HAXM" or "Android Emulator Hypervisor Driver for AMD Processors",
+> you can safely ignore them as they won't be used for skyline
+
+</p>
+</details>
+
+Import the project: click on the `Open` icon then in the file picker choose the `skyline` folder you just cloned in the steps above
```
[the remark this answers is not in the document]
Comment (lynxnb) [reply BUILDING.md:39]: I'll reword the sentence a bit to remove the colon

**Comment by greggameplayer** [comment]
@lynxnb Why not doing a wiki page for the building guide instead of a markdown file ?

**Comment by PixelyIon** [comment]
@greggameplayer We like documentation like that being more front and center, it being available locally on being cloned is another reason. Either way, we're unlikely to change this.

**Review by PixelyIon — APPROVED** [review]
This looks really nice, good job!

**Pull request merged**